src/helper.ts

```diff
@@ -22,6 +22,7 @@ import {
   IdentifierDotExpressionCstNode,
   IndexExpressionCstNode,
 } from './cst-definitions.js'
+import { equals } from 'ramda'
```

Comment:
`assert` was node specific, I replaced it to ramda's `equals`. This solved the issue and simplified code

Reply:
perfect!!!